packages/rainbowkit/src/hooks/useMainnetEnsName.ts

```diff
@@ -1,13 +1,10 @@
+import { mainnet } from 'viem/chains';
 import { useEnsName } from 'wagmi';
-import { useMainnet } from './useMainnet';
 
 export function useMainnetEnsName(address: string | undefined) {
-  const { chainId, enabled } = useMainnet();
-
   const { data: ensName } = useEnsName({
-    address,
-    chainId,
-    enabled,
+    chainId: mainnet.id,
+    address: address as `0x`,
```

Comment:
can't we have the arg type as `Address` instead of casting here?
```diff
export function useMainnetEnsName(
- address: string | undefined
+ address: Address | undefined
) {
```

Reply:
Oh yeah we can. I guess we can import it from `viem`, but good catch 🙏 🙌